news/_posts/2020-05-07-productboard-portal-launch.md

```diff
@@ -0,0 +1,100 @@
+
+So the team has created this portal called **PrestaShop Feedback** so that anyone can comment and vote for the features on which feedback is needed. Whether you are a merchant, a developer, an agency, a seller on the [Addons Marketplace](http://addons.prestashop.com/), etc. your opinions and suggestions are expected on the [feedback.prestashop.com](https://feedback.prestashop.com) page.
+
+The portal, available in the 'PrestaShop Core' tab, is divided into several sections. The goal of this article is to explain how to use this interface to share your feedback on upcoming features.
+
+
+## How to use PrestaShop Feedback
+
+### See the selected features, express some needs, check their design
+
+Each feature or expression of needs is represented by a card that includes a title and a description, sometimes an image.
+Click on a card to learn more about it.
+
+All cards fall into three sections, each corresponding to a specific step in the product development:
+
+- Under consideration
+- User research
+- Design
+
+![PrestaShop Feedback home](/assets/images/2020/05/productboard-home.png)
+
+#### Under consideration
+Lists the topics and features that have already been identified and structured by the team. At this step, we need feedback to consider as many use cases as possible. It will allow us to draw the most useful feature. Also, rating a card to let the team know how important it is to you will help us prioritize it.
+
+#### User research
+Lists the features the team decided to work on. When we start working on a feature, the first step is always to drive some user research to understand which types of users it may concern, what are the use cases around it, and its purpose. As the goal of user research is to identify the problem or need to solve, the purpose at this point is to get feedback to consider which problems are related to which cases.
+
+#### Design
+Once the problem and the solutions identified, the design phase is here to build the feature. In this section, you will be able to find all the features that are going under design. Getting feedback allows the team to know how useful is the solution and to possibly recruit voluntary testers. When the design is approved, the development phase goes back to [GitHub](https://github.com/PrestaShop/PrestaShop).
+
+
+### Comment and vote for the cards
+
+When clicking on a card, you can see two fields at its bottom: one to comment and another one to vote. It is really important to the team as it helps us evaluate how the feature impacts **you**. Here is what happens behind the scenes:
+
+![PrestaShop Productboard user score](/assets/images/2020/05/productboard-user-score.png)
+
+When rating the impact of a feature, Productboard adds your vote to a “user score” that the team takes directly into account when evaluating features through its prioritization score.
+The more an idea is voted, the better rank it gets, and the more probability it has to integrate the software.
```

Comment:
ranking ?

Reply:
rather than integrate the software, i'd say "to be developed, either by our teams or by community developers".
Parce qu'il y a sûrement des features qu'on ne pourra pas développer (quand bien même elles sont prioritaires), et qu'on poussera à la communauté